packages/next/src/client/components/react-dev-overlay/internal/styles/Base.tsx

```diff
@@ -18,12 +18,12 @@ export function Base() {
           --size-font-bigger: 24px;
 
           --color-background: white;
-          --color-font: #757575;
+          --color-font: #444;
           --color-backdrop: rgba(17, 17, 17, 0.2);
 
           --color-stack-h6: #222;
-          --color-stack-headline: #666;
-          --color-stack-subline: #999;
+          --color-stack-headline: #555;
+          --color-stack-subline: #585858;
```

Comment:
Slightly improved color contrast to pass basic a11y requirements. It could probably still be improved by a proper redesign (out of scope for this PR)